consensus/ethash/algorithm_go1.8.go

```diff
@@ -48,7 +52,10 @@ func datasetSize(block uint64) uint64 {
 	if epoch < len(datasetSizes) {
 		return datasetSizes[epoch]
 	}
-	// No known dataset size, calculate manually (sanity branch only)
+	return calcDatasetSize(epoch)
+}
+
+func calcDatasetSize(epoch int) uint64 {
```

Comment:
Please document this method. I know it's private, but all methods in ethash are fully documented and I'd like to keep it so.